edenia/antelope-tools
# Feat/dark mode 497

### Dark mode implementation

### What does this PR do?

- Resolve #497
- Resolve #1352
- Resolve #1351

### Steps to test

1. Run the project locally
2. Check the dark mode
3. Check the new texts

### Screenshots

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/b5e63367-5849-457e-acd5-fa849f652eed)

# Block Producers

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/3d319ffe-d7b4-4703-9ba2-4f0215cb86db)

# BP Profile

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/3fc1193b-d332-4e91-95ee-ec2f5f85f7e0)

# Nodes Distribution

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/cdc88a72-4c42-4803-8a8c-f991f90beabe)

# Rewards Distribution

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/d624b2c2-0532-401f-b210-f8fa3e7b72f0)

# Blocks Distribution

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/e1f20cee-ca10-45c3-9b43-a192a1079e0e)

# Ricardian Contract

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/07fc6969-9fe9-42bd-a1f7-de8af95035b5)

# EVM endpoints

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/a3d7fc3e-9a63-4655-aad7-55c50a194618)

# New titles and descriptions

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/09a599cc-2c80-48ac-afa6-c7d006242fff)
